### ISSUE TITLE
Unstable training loss

### ISSUE THREAD

**Issue by LiuBodan** [crosszamirski/WS-DINO#6]
Hi, 
I tried to reproduce the model, however, my training loss is very unstable:
![image](https://github.com/crosszamirski/WS-DINO/assets/123052690/bb6cebad-28ef-4d9d-8a60-e3ad8719b788)

For preprocessing:

I downloaded the dataset from the BBBC021 official website, then used the two CellProfiler pipelines provided to produce the training data, where I preprocessed annotated data with DMSO data together. (Should I do the preprocessing for annotated and DMSO separately?)

I trained the model with the same parameters on 2x Tesla V100 using "python -m torch.distributed.launch --nproc_per_node=2 WS-DINO_BBBC021.py"

Could anyone point out where I did it wrong?

Many thanks
